_maps/RandomRuins/LavaRuins/lavaland_surface_dorffort.dmm

```diff
@@ -0,0 +1,641 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"a" = (
+/turf/template_noop,
+/area/template_noop)
+"b" = (
+/turf/closed/mineral/random/volcanic,
+/area/mine/explored)
+"c" = (
+/obj/machinery/anvil,
+/obj/item/weapon/smith_hammer,
+/turf/open/floor/plating/asteroid/basalt/lava_land_surface,
+/area/awaymission)
```

Comment:
These should probably have their own areas instead of /area/awaymission. If someone built an APC on one of them, it'd affect all of them since they have the same path.